packages/backend/src/controllers/house/house.controller.ts

```diff
@@ -75,13 +88,23 @@ export class HouseController {
     const userDoc = await this.userStoreService.findOneByFirebaseId(user.uid);
     if (userDoc.house != undefined) {
       const house = await this.houseStoreService.findOne(userDoc.house);
+      const userList: Array<UserResponseDto> = [];
+      for (const id of house.users) {
+        const user = await this.userStoreService.findOne(id);
+        const userDto = {
+          house: house.code,
+          firebaseId: user.firebaseId,
+        };
+        userList.push(userDto);
+      }
```

Comment:
Could we use populate instead of this? This would need to be done inside the houseStoreService. 
https://mongoosejs.com/docs/populate.html#population

Reply:
In this way we would need to inject the userStoreService to the houseStoreService to get the user right? Which will eventually end up causing errors in house service test.

Reply:
you shouldn't need to inject the service. Please take a look at the docs that @mped822 linked